website/server/controllers/api-v3/groups.js

```diff
@@ -669,38 +674,68 @@ async function _inviteByEmail (invite, group, inviter, req, res) {
+ * @apiParam {String} [emails.name] The name of the user being invited.
  * @apiParam {Array} [uuids] Body parameter - An array of uuids to invite
  *
  * @apiParamExample {json} Emails
- *   {
- *     "emails": ["user-1@example.com", "user-2@exmaple.com"]
- *   }
+ * {
+ *    "emails": [
+ *      {"email": "user-1@example.com"},
+ *      {"name": "User2", "email": "user-2@example.com"}
+ *    ]
+ *}
  * @apiParamExample {json} User Ids
  *   {
  *     "uuids": ["user-id-of-existing-user", "user-id-of-another-existing-user"]
  *   }
  * @apiParamExample {json} User Ids and Emails
- *   {
- *     "emails": ["user-1@example.com", "user-2@exmaple.com"],
- *     "uuids": ["user-id-of-existing-user", "user-id-of-another-existing-user"],
- *   }
+ * {
+ *   "emails": [
+ *       {"email": "user-1@example.com"},
+ *       {"email": "user-2@example.com"}
+ *   ],
+ *   "uuids": ["user-id-of-existing-user"]
+ *}
  *
  * @apiSuccess {Array} data The invites
  * @apiSuccess {Object} data[0] If the invitation was a user id, you'll receive back an object. You'll recieve one Object for each succesful user id invite.
  * @apiSuccess {String} data[1] If the invitation was an email, you'll receive back the email. You'll recieve one String for each successful email invite.
  *
- * @apiSuccessExample {json} Successful Response
- *   {
- *     data: [
- *       { id: 'the-id-of-the-invited-user', name: 'The group name', inviter: 'your-user-id' },
- *       "user@example.com"
+ * @apiSuccessExample {json} Successful Response with Emails
+ * {
+ *   "data": [
+ *       "user-1@example.com",
+ *       "user-2@exmaple.com"
+ *   ]
+ *}
+ *
+ * @apiSuccessExample {json} Successful Response with User Id
+ * {
+ *     "data": [
+ *       { id: 'the-id-of-the-invited-user', name: 'The group name', inviter: 'your-user-id' }
  *     ]
- *   }
+ * }
  *
+ * @apiUse GroupBodyInvalid
  * @apiUse GroupNotFound
- * @apiError InvalidInvitationParams An error relating to the data sent in `emails` and/or `uuids`
- * @apiError TooManyInvites A max of 100 invites (combined emails and user ids) can be sent out at a time
+ * @apiError (400) {BadRequest} NoEmailProvided An email address was not provided in the `emails` body param
+ * @apiUse PartyNotFound
+ * @apiError (400) {BadRequest} UuidOrEmailOnly The `emails` and `uuids` params
+ * were both missing and/or a key other than `emails` or `uuids` was provided in
+ * the body param.
+ * @apiError (400) {BadRequest} CannotInviteSelf User id or email of invitee matches that
+ * of inviter.
+ * @apiError (400) {BadRequest} UserAlreadyInvited The user has already been
+ * invited to the group.
+ * @apiError (400) {BadRequest} UserAlreadyInGroup The user is already a member
+ * of the group.
+ * @apiError (400) {BadRequest} MustBeArray The `uuids` or emails body param is
+ * not an array.
+ * @apiError (400) {BadRequest} UserNotFound One or more of the `uuids` does not
```

Comment:
It's actually a `404 NotFound` error. You can just use the `UserNotFound` error defined in the user controller.

```
@apiUse UseNotFound
```